src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -741,52 +379,26 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         return 0;
     }
 
-    // Method Description:
-    // - Ask the monarch to show a notification icon.
-    // Arguments:
-    // - <none>
-    // Return Value:
-    // - <none>
-    winrt::fire_and_forget WindowManager::RequestShowNotificationIcon()
-    {
-        co_await winrt::resume_background();
-        _peasant.RequestShowNotificationIcon();
-    }
-
-    // Method Description:
-    // - Ask the monarch to hide its notification icon.
-    // Arguments:
-    // - <none>
-    // Return Value:
-    // - <none>
-    winrt::fire_and_forget WindowManager::RequestHideNotificationIcon()
-    {
-        auto strongThis{ get_strong() };
-        co_await winrt::resume_background();
-        _peasant.RequestHideNotificationIcon();
-    }
-
     // Method Description:
     // - Ask the monarch to quit all windows.
     // Arguments:
     // - <none>
     // Return Value:
     // - <none>
-    winrt::fire_and_forget WindowManager::RequestQuitAll()
+    winrt::fire_and_forget WindowManager::RequestQuitAll(Remoting::Peasant peasant)

```

Comment:
nit: copy?

Reply:
I mean, it's on a bg thread, so it needs to be a copy?